packages/react-components/src/shared/src/useChainedEventCallback.ts

```diff
@@ -0,0 +1,19 @@
+// Copied from https://github.com/react-bootstrap/react-bootstrap/blob/master/src/createChainedFunction.js.
+
+import { useEventCallback } from "./useEventCallback";
+
+type ChainableFunction = (...args: any[]) => void;
+
+export function createChainedFunction<T extends ChainableFunction>(...funcs: (T | false)[]) {
+    return (...args: any[]) => {
+        (funcs
+            .filter(Boolean) as T[])
```

Comment:
Not a big deal but I have seeing parantheses starting and finishing on a different line like this!